extensions/ql-vscode/src/bqrs-types.ts

```diff
@@ -0,0 +1,102 @@
+/**
```

Comment:
There is some duplicates between this file and `bqrs-util-types.ts`. This doesn't need to be addressed now, but we should add a TODO or create an issue for that.

Reply:
It's worse than that --- the overlap between the two files is not exact duplication, but a bunch of pairs of types that sometimes share the same name and purpose, and which aren't exactly the same. This is exactly the migration work that remains to be done, but which can be done independently of removing the other build infrastructure.

I added the word `TODO` to the comment at the top of `bqrs-types.ts`, at least, if that helps :)